dog.py

```diff
@@ -445,6 +515,8 @@ def docker_run(config: DogConfig) -> int:
     args += [
         'run',
         '--rm',
+        '--network',
```

Comment:
I have a feeling that it is better to leave out --network unless the user specifies it explicitly in the dog.config file - what if the defaults changes in the future for some unknown reason - or is different on one platform vs. another.

Reply:
Using --network host or --network none gives a 0,5 s faster startup - it's almost half the time (!!)
I have chosen host as default (if not specified by dog.config). I'm also OK with leaving this out and only setting it if network is specified in dog.config but I felt that the speedup is something you always want.